content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/150-introspection.mdx

```diff
@@ -497,24 +497,26 @@ The data model now looks similar to this (note that the fields on the models hav
 model Post {
   id        Int      @id @default(autoincrement())
   title     String   @db.VarChar(255)
-  createdAt DateTime @default(now()) @db.Timestamp(6)
-  content   String?
+  createdAt DateTime @default(now()) @db.Timestamp(0)
```

Comment:
Is that `0` correct?

Reply:
Yes. I created a new DB, pasted the SQL for the schema and ran introspection. That's what I got back.

Reply:
Can this be migrated and does it work?

Reply:
Yes and yes.

I followed the guide, and it generated the following migration using `prisma migrate diff` and baseline it.

```sql
-- CreateTable
CREATE TABLE `Post` (
    `id` INTEGER NOT NULL AUTO_INCREMENT,
    `title` VARCHAR(255) NOT NULL,
    `createdAt` TIMESTAMP(0) NOT NULL DEFAULT CURRENT_TIMESTAMP(0),
    `content` TEXT NULL,
    `published` BOOLEAN NOT NULL DEFAULT false,
    `authorId` INTEGER NOT NULL,

    INDEX `authorId`(`authorId`),
    PRIMARY KEY (`id`)
) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;

-- CreateTable
CREATE TABLE `Profile` (
    `id` INTEGER NOT NULL AUTO_INCREMENT,
    `bio` TEXT NULL,
    `userId` INTEGER NOT NULL,

    UNIQUE INDEX `userId`(`userId`),
    PRIMARY KEY (`id`)
) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;

-- CreateTable
CREATE TABLE `User` (
    `id` INTEGER NOT NULL AUTO_INCREMENT,
    `name` VARCHAR(255) NULL,
    `email` VARCHAR(255) NOT NULL,

    UNIQUE INDEX `email`(`email`),
    PRIMARY KEY (`id`)
) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;

-- AddForeignKey
ALTER TABLE `Post` ADD CONSTRAINT `Post_ibfk_1` FOREIGN KEY (`authorId`) REFERENCES `User`(`id`) ON DELETE NO ACTION ON UPDATE NO ACTION;

-- AddForeignKey
ALTER TABLE `Profile` ADD CONSTRAINT `Profile_ibfk_1` FOREIGN KEY (`userId`) REFERENCES `User`(`id`) ON DELETE NO ACTION ON UPDATE NO ACTION;
```

I ran a query and the response seemed to contain the right format.

```
[
  {
    id: 1,
    title: 'Hello World',
    createdAt: 2023-04-28T13:04:09.000Z,
    content: null,
    published: false,
    authorId: 1
  }
]
```

Is this not expected to work/ run? Should I raise this issue with the schema team? 🤔

Reply:
Ah, I see what the issue is. The precision of the timestamp is set to 0 (seconds) 😅 

Should this be a bug in Prisma Migrate?

Reply:
I've started a conversation internally. If this works as it should, I'll update the generated SQL in the baselining chapter.